cmake/msvc.cmake

```diff
@@ -1,11 +1,21 @@
-
-FUNCTION(kokkos_set_msvc_flags full_standard int_standard)
-  IF (CMAKE_CXX_EXTENSIONS)
-    SET(KOKKOS_CXX_STANDARD_FLAG "" PARENT_SCOPE)
-    SET(KOKKOS_CXX_INTERMEDIATE_STANDARD_FLAG "" PARENT_SCOPE)
-  ELSE()
-    SET(KOKKOS_CXX_STANDARD_FLAG "" PARENT_SCOPE)
-    SET(KOKKOS_CXX_INTERMEDIATE_STANDARD_FLAG "" PARENT_SCOPE)
-  ENDIF()
-ENDFUNCTION()
-
+function(kokkos_set_msvc_flags full_standard int_standard)
+  if(CMAKE_CXX_EXTENSIONS)
+    set(KOKKOS_CXX_STANDARD_FLAG
+        ""
+        PARENT_SCOPE
+    )
+    set(KOKKOS_CXX_INTERMEDIATE_STANDARD_FLAG
+        ""
+        PARENT_SCOPE
+    )
```

Comment:
The previous version was better to me...

Reply:
I agree.